libs/ldmk-transport/src/index.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
+import Transport from "@ledgerhq/hw-transport";
+import {
+  DeviceSdkBuilder,
+  ConsoleLogger,
+  type DeviceSdk,
+  type DeviceSessionState,
+  DeviceStatus,
+  LogLevel,
+} from "@ledgerhq/device-management-kit";
+import { BehaviorSubject, firstValueFrom } from "rxjs";
+
+const deviceSdk = new DeviceSdkBuilder().addLogger(new ConsoleLogger(LogLevel.Debug)).build();
```

Comment:
The dmdk transport integration dmdk is available on develop.
You should use the latest version and `builder.addCustomTransport(DMDKTransport)` here with a transport defined as `class DMDKTransport extends Transport`

Reply:
@jdabbech-ledger it's a LL `Transport` here not a LDMK transport

Reply:
It's a transport like other ledger live transports, but it's a wrapper around the LDMK.

![Screenshot 2024-09-26 at 12 02 07](https://github.com/user-attachments/assets/3149e84b-a54a-4be7-aa20-ce57cc067331)

Reply:
Oh ok, jira leaks information on this part. 
Will you add here a node hid transport afterwards?